source/module_hamilt_general/module_xc/xc_functional_vxc.cpp

```diff
@@ -10,7 +10,17 @@
 #include "module_parameter/parameter.h"
 
 #ifdef USE_LIBXC
+#include "NCLibxc/NCLibxc.h"
 #include "xc_functional_libxc.h"
+#include <tuple>
```

Comment:
Why do you add so many "include" but not use them?

Reply:
It is for future development of this package and more flexible interfaces. We don't submit those parts to ABACUS. If they are annoying, it is OK to delete them.